src/server/routes/configuration/wallets/get.ts

```diff
@@ -43,10 +37,23 @@ export async function getWalletsConfiguration(fastify: FastifyInstance) {
         [StatusCodes.OK]: responseBodySchema,
       },
     },
-    handler: async (req, res) => {
+    handler: async (_req, res) => {
```

Comment:
Any reason for this change? Looks unneeded

Reply:
biome complains if any variables are unused, unless they are underscore prefixed.

I'm trying to make every file I touched lint-error free